octodns/provider/base.py

```diff
@@ -58,7 +60,7 @@ def plan(self, desired):
 
         # allow the provider to filter out false positives
         before = len(changes)
-        changes = filter(self._include_change, changes)
+        changes = list(filter(self._include_change, changes))
```

Comment:
Let’s make this a __list comprehension__ as discussed at https://docs.python.org/3/library/functions.html#filter

Reply:
I'm not sure what you mean.

Reply:
The above link didn't really point me in the right direction, but https://portingguide.readthedocs.io/en/latest/iterators.html#higher-order-functions-vs-list-comprehensions did.